Interop/ModelCard/src/main/java/org/tribuo/interop/modelcard/package-info.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * ModelCard feature to allow more transparent model reporting.
```

Comment:
I think it might be nice to paste in an example model card in json format here so people get the idea, along with a citation to the model card paper.